src/odin/ingestion/afc/afc_archive.py

```diff
@@ -295,11 +306,12 @@ def run(self) -> int:
         """
         self.start_kwargs = {"table": self.table}
         # TODO: Removed cert_reqs parameter when not using self-signed endpoint.
-        # Current timetout set to 15 mins, for full PROD deployment should be no more than 5 mins.
+        # Current timetout set to 10 mins, for full PROD deployment should be no more than 2 mins.
         self.req_pool = urllib3.PoolManager(
             headers=self.headers,
             cert_reqs="CERT_NONE",
-            timeout=urllib3.Timeout(total=60 * 15),  # 15 minute total timeout
+            timeout=urllib3.Timeout(total=60 * 10),  # 10 minute total timeout
+            retries=False,  # No retires, if retry-able failures are documented, can be updated
```

Comment:
Doesn't seem like we ever get anything back after 10 minutes, so just reduce this done and disable retries. There might be legitimate reasons to retry in the future, but those situations will be pulled from logging and explicitly added for retry logic.